arches/install/arches-templates/tests/test_settings.py-tpl

```diff
@@ -92,38 +69,10 @@ LOGGING["loggers"]["arches"]["level"] = "ERROR"
 ELASTICSEARCH_PREFIX = "test"
 
 TEST_RUNNER = "tests.base_test.ArchesTestRunner"
-SILENCED_SYSTEM_CHECKS.append("arches.W001")  # Cache backend does not support rate-limiting
-
-# could add Chrome, PhantomJS etc... here
-LOCAL_BROWSERS = []  # ['Firefox']
-
-ENABLE_USER_SIGNUP = True
-FORCE_USER_SIGNUP_EMAIL_AUTHENTICATION = True
-
-OVERRIDE_RESOURCE_MODEL_LOCK = True
-
-ENABLE_TWO_FACTOR_AUTHENTICATION = False
-FORCE_TWO_FACTOR_AUTHENTICATION = False
+SILENCED_SYSTEM_CHECKS.append(
+    "arches.W001",  # Cache backend does not support rate-limiting
+)
 
-DATATYPE_LOCATIONS.append("tests.fixtures.datatypes")
-ELASTICSEARCH_HOSTS = [{"scheme": "http", "host": "localhost", "port": ELASTICSEARCH_HTTP_PORT}]
-LANGUAGES = [
-    ("de", _("German")),
-    ("en", _("English")),
-    ("en-gb", _("British English")),
-    ("es", _("Spanish")),
-    ("ar", _("Arabic")),
+ELASTICSEARCH_HOSTS = [
+    {"scheme": "http", "host": "localhost", "port": ELASTICSEARCH_HTTP_PORT}
 ]
-
-DOCKER = False
-
-try:
-    from arches.settings_local import *
-except ImportError:
-    pass
-
-if DOCKER:
-    try:
-        from arches.settings_docker import *
-    except ImportError:
-        pass
```

Comment:
Project level settings will have already done these imports, so doing them again here at the end risks introducing bugs.